docs/email.md

```diff
@@ -31,39 +31,128 @@ SMTP_TIMEOUT
 
 This is an example of a configuration using [Mailgun](https://www.mailgun.com/) email service:
 
+```yaml
+- SMTP_HOST=smtp.eu.mailgun.org
+- SMTP_PORT=465
+- SMTP_TLS=true
+- SMTP_USERNAME=postmaster@mg.example.com
+- SMTP_PASSWORD=secretpassword
+- AUTH_EMAIL_FROM=notify@example.com
+- NOTIFY_EMAIL_FROM=notify@example.com
 ```
-      - SMTP_HOST=smtp.eu.mailgun.org
-      - SMTP_PORT=465
-      - SMTP_TLS=true
-      - SMTP_USERNAME=postmaster@mg.example.com
-      - SMTP_PASSWORD=secretpassword
-      - AUTH_EMAIL_FROM=notify@example.com
+
+#### API
+
+Using SMTP directly as in example above exposes sending server IP address. Alternative
```

Comment:
I think the part about "exposes sending server IP address" should be clarified a little bit. As far as I recall this "exposing" is a part of one of the SMTP RFCs and not a side effect of the particular mailgun relay usage.

Reply:
Clarified a little, please check the current version.